capstone-ui/src/Components/StateMap/StateMap.css

```diff
@@ -2,4 +2,11 @@
     fill: #e3e3e3;
     stroke: #333;
     stroke-width: 0.5;
+}
+
```

Comment:
I think this commit could have been part of previous as last commit introduced the styling for `.popup-container`.

Reply:
I'll keep that in mind